cmd/blockstorage_attach.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/exoscale/cli/pkg/globalstate"
+	"github.com/exoscale/cli/pkg/output"
+	v3 "github.com/exoscale/egoscale/v3"
+)
+
+type blockstorageAttachCmd struct {
```

Comment:
Please rename all these command types to use camelCase
blockstorage -> blockStorage

Reply:
https://github.com/exoscale/cli/pull/574/commits/8ad84de6bb941b377e66d6a4287655a94055e898